Win11ThemeTest/CalendarTest.cs

```diff
@@ -0,0 +1,475 @@
+﻿using FlaUI.Core;
+using FlaUI.Core.AutomationElements;
+using FlaUI.Core.Input;
+using FlaUI.UIA3;
+using FlaUI.Core.Definitions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Globalization;
+using Calendar = FlaUI.Core.AutomationElements.Calendar;
+using System.Configuration;
+
+namespace Win11ThemeTest
+{
+    public class CalendarTest
+    {
+        private Application app;
+        private Window window;
+        public Window calwindow;
+        Button testButton;
+        Calendar calendar;
+        Calendar multiSelectCalendar;
+        AutomationElement headerBtn;
+        AutomationElement prevBtn;
+        AutomationElement nextBtn;
+        public CalendarTest()
+        {
+            try
+            {
+
+                //app = Application.Launch(@"..\\..\\..\\..\\TestingApplication\\bin\\Debug\\net9.0-windows\\win-x64\\TestingApplication.exe");
+                var appPath = ConfigurationManager.AppSettings["Testpath"];
+                app = Application.Launch(appPath);
+                using (var automation = new UIA3Automation())
+                {
+                    window = app.GetMainWindow(automation);
+                    testButton = window.FindFirstDescendant(cf => cf.ByAutomationId("calendartestbtn")).AsButton();
+                    Mouse.Click(testButton.GetClickablePoint());
+                    Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(2000));
+                    calwindow = window.FindFirstDescendant(cf => cf.ByName("CalendarWindow")).AsWindow();
+                    calendar = calwindow.FindFirstDescendant(cf => cf.ByAutomationId("tstCal")).AsCalendar();
+                    multiSelectCalendar = calwindow.FindFirstDescendant(cf => cf.ByAutomationId("tstCal_multiSelect")).AsCalendar();
+                }
+            }
+            catch (Exception ex)
+            {
+                var filepath = ConfigurationManager.AppSettings["logpath"];
+                if (!Directory.Exists(filepath))
+                {
+                    Directory.CreateDirectory(filepath);
```

Comment:
Address this 

![image](https://github.com/singhashish-wpf/Win11ThemeSampleApp/assets/85492722/93662ff1-8d4d-4d13-ba5f-12fe3a59e709)